### DISCUSSION TITLE
Feed is...Invisible?

### DISCUSSION THREAD

**Discussion by adamesq** [rrirower/Channel-Feed-Manager#116] [Q&A] [answered]
This one could be a limitation of my Windows knowledge. When I try to open a feed, it appears to be invisible...or in what we would call a virtual desktop on a Mac. However, no images show. I think the image will explain what I'm probably failing to explain here...

<img width="977" alt="Screenshot 2023-11-02 at 5 00 34 PM" src="https://github.com/rrirower/Channel-Feed-Manager/assets/34196956/f22b19b2-30ed-46d2-be40-77c1d69189d7">

HeLp please!

**Comment by rrirower** [comment]
I'm not sure what the issue could be. You can try uninstalling and reinstalling.  But, it also looks like it's some kind of framework issue. I think you said you're running Windows in a virtual machine. Maybe there's an issue there.

**Reply by adamesq** [reply]
Thanx for the reply. Must be on the Windows side then. Which files do I need to save the library through the un/reinstall?

**Reply by rrirower** [reply, marked as answer]
Consider saving "C:\Users\XXXX\Documents\Channel Feed Manager" although the channel stuff is in "Channels" sub-folder.

**Reply by adamesq** [reply]
Something definitely went wacky in the AWS Windows instance. I created another one and got this back up and running. Thank you! I've got some catching up to do on recent updates and posts from others.
